src/ol/featureloader.js

```diff
@@ -87,8 +87,10 @@ ol.featureloader.tile = function(url, format) {
        * @this {ol.VectorTile}
        */
       function(features, dataProjection) {
-        this.setProjection(dataProjection);
-        this.setFeatures(features);
+        /** @type {ol.VectorTile} */
+        var me = this;
```

Comment:
Thanks @ahocevar for the suggestion. Unfortunately I just tried it but I got the same error.